lib/client.ts

```diff
@@ -691,6 +768,136 @@ export class Client extends EventEmitter {
     this.jsFramework = detectFramework();
   }
 
+  private getUsernameFromToken = (parsedToken: string | any): string => {
+    if (parsedToken['sub']) {
+      return parsedToken['sub'];
+    }
+    const randomTenDigitNumber = Math.floor(Math.random() * 10000000000);
+    return "puser" + randomTenDigitNumber.toString() + "jt";
+  };
+
+  private validateToken = (parsedToken: string | any): boolean => {
+
+    if (parsedToken == null) {
+      return false;
+    }
+
+    let { app = undefined, lss = undefined, sub = undefined, nbf = undefined, exp = undefined, per = undefined } = parsedToken;
+    let { incoming_allow = undefined, outgoing_allow = undefined } = per.voice || undefined;
+  
+    // To do : Add the token validations [DONE]
+    if(!lss || !nbf || !exp || !per || incoming_allow === undefined || outgoing_allow === undefined) {
+      return false;
+    }
+    
+    if(typeof app !== "string" || typeof lss !== "string" || typeof sub !== "string" || typeof nbf !== "number" || 
```

Comment:
its not lss. it should be iss. please correct it everywhere

Reply:
the "lss" parameter I received from token when I parse it

Reply:
@gaurav-plivo It is mentioned lss it has to be is in line 793